app/src/pages/authoring/[resourceType]/[id].tsx

```diff
@@ -58,21 +70,55 @@ export default function ResourceAuthoringPage() {
     }
   });
 
-  function parseUpdate(url: string, identifier: string) {
-    const update: { url?: string; identifier?: fhir4.Identifier[] } = {};
+  function parseUpdate(url: string, identifier: string, name: string, title: string, description: string) {
+    const additions: {
+      url?: string;
+      identifier?: fhir4.Identifier[];
+      name?: string;
+      title?: string;
+      description?: string;
+    } = {};
+
+    const deletions: {
+      url?: string;
+      identifier?: fhir4.Identifier[];
+      name?: string;
+      title?: string;
+      description?: string;
+    } = {};
+
     if (url !== '') {
-      update['url'] = url;
+      additions['url'] = url;
+    } else {
+      deletions['url'] = '';
     }
```

Comment:
This might be an odd point, but if the user were to add a bunch of blank spaces or even just one space (i.e. " ") for an attribute, the attribute's value gets set to the blank space/s. Would it be worthwhile to modify this so that a character of some sort must be entered into the TextInput in order for the attribute's value to be assigned?

Reply:
Good catch & idea!